core/src/main/java/org/geysermc/geyser/entity/type/living/ArmorStandEntity.java

```diff
@@ -362,7 +362,7 @@ private void updateSecondEntityStatus(boolean sendMetadata) {
             setFlag(EntityFlag.INVISIBLE, false);
             dirtyMetadata.put(EntityData.SCALE, 0.0f);
             // As the above is applied, we need an offset
-            updateOffsetRequirement(true);
+            updateOffsetRequirement(false);
```

Comment:
This seems to fix the passenger's position, however it causes the name tag of the armor stand to appear far too low.